gradle/libs.versions.toml

```diff
@@ -2,7 +2,12 @@
 kotlin = "1.8.10"
 coroutines = "1.6.4"
 
-java = "11"
+java-target= "11"
+java-toolchain = "11"
+java-gradle = "11"
```

Comment:
I'd leave this out for opensource projects, we don't need to force a specific version for running Gradle
```suggestion
```